File: C/1010.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int codigo_peca1, qtde_peca1, codigo_peca2, qtde_peca2;
    float valor_peca1, valor_peca2, valor_total;

    scanf("%d %d %f", &codigo_peca1, &qtde_peca1, &valor_peca1);
    scanf("%d %d %f", &codigo_peca2, &qtde_peca2, &valor_peca2);

    valor_total = (float)(qtde_peca1 * valor_peca1) + (float)(qtde_peca2 * valor_peca2);

    printf("VALOR A PAGAR: R$ %.2f\n", valor_total);
    return 0;
}

File: Pyhon/1010.py
lista1 = input().split(" ")
codigo_peca1 = int(lista1[0])
qtde_peca1 = int(lista1[1])
valor_unit_peca1 = float(lista1[2])

lista2 = input().split(" ")
codigo_peca2 = int(lista2[0])
qtde_peca2 = int(lista2[1])
valor_unit_peca2 = float(lista2[2])

valor_total = ((qtde_peca1*valor_unit_peca1) + (qtde_peca2*valor_unit_peca2))

print("VALOR A PAGAR: R$ %.2f" %valor_total)